### PR TITLE
New package: python-pyaes-1.6.0

### PR DESCRIPTION
**Pull request by ananteris**

### CODE REVIEW

File: srcpkgs/python-pyaes/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+pkgname=python-pyaes
+version=1.6.0
+revision=1
+noarch=yes
+wrksrc="pyaes-${version}"
+build_style=python2-module
+hostmakedepends="python-setuptools"
+depends="python"
+pycompile_module="pyaes.py"
+short_desc="Pure-Python Implementation of the AES block-cipher and common modes of operation"
```
Comment (pullmoll) [inline srcpkgs/python-pyaes/template]: I think `short_desc="Pure Python implementation of the AES block-cipher"` would suffice. See xlint warning about the length of short_desc being > 72.

File: srcpkgs/python-pyaes/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+maintainer="ananteris <ananteris@protonmail.ch>"
+homepage="https://pypi.python.org/pypi/pyaes"
+license="MIT"
```
Comment (pullmoll) [inline srcpkgs/python-pyaes/template]: `license="MIT"` requires a `post_install()` with `vlicense LICENSE.txt`.

File: srcpkgs/python-pyaes/template (hunk 1 above)
Comment (ananteris) [inline srcpkgs/python-pyaes/template]: Fixed

File: srcpkgs/python-pyaes/template (hunk 2 above)
Comment (ananteris) [inline srcpkgs/python-pyaes/template]: Fixed

File: srcpkgs/python-pyaes/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'python-pyaes'
+pkgname=python-pyaes
+version=1.6.0
+revision=1
+noarch=yes
+wrksrc="pyaes-${version}"
+build_style=python2-module
+hostmakedepends="python-setuptools"
+depends="python"
+pycompile_module="pyaes.py"
```
Comment (asergi) [inline srcpkgs/python-pyaes/template]: `pycompile_module="pyaes"`

File: srcpkgs/python-pyaes/template [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'python-pyaes'
+pkgname=python-pyaes
```
Comment (asergi) [inline srcpkgs/python-pyaes/template]: Add python3 flavor. See `python-sh` as example.

File: srcpkgs/python-pyaes/template (hunk 3 above)
Comment (ananteris) [inline srcpkgs/python-pyaes/template]: @asergi what is the significance of this? there are 310 packages already in the repo that have a template that uses pycompile_module="${module}.py" entry.. should all of these be updated?

Comment (ananteris) [inline srcpkgs/python-pyaes/template]: I see what the difference is now, infact python-pycrypto should not be it's own package, it's already provided by python-crypto. I've removed it.

**Comment by asergi** [comment]
fcee34f61a3a0756eba93641443f190578bbe3f3

**Pull request closed without merging**